backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlSessionFieldFactory.java

```diff
@@ -117,6 +122,7 @@ public GraphQLFieldDefinition userQueryField(Database database, Schema schema) {
               if (schema != null) {
                 result.put(ROLES, schema.getInheritedRolesForActiveUser());
               }
+              result.put(SCHEMAS, database.getSchemaNames());
```

Comment:
all schemas or all readable by user ?

Reply:
this is automatically filtered to only show schemas available to user